devices/System.Diagnostics.Stopwatch/Stopwatch.cs

```diff
@@ -53,7 +53,7 @@ public Stopwatch()
         /// Gets the current number of ticks in the timer mechanism.
         /// </summary>
         /// <returns></returns>
-        public static long GetTimestamp() => Environment.TickCount64;
+        public static long GetTimestamp() => Environment.TickCount64 * TimeSpan.TicksPerMillisecond;
```

Comment:
Suggestion unrelated to this PR: adding a timestamp in ms ?
I don't know how much ticks are used on microcontrollers but so far I've often - if not always - seen ms for this kind of thing. Maybe ticks are more reliable than ms on microcontrollers ?

Reply:
Ticks in MCU RTOS are different then [.NET ticks](https://docs.microsoft.com/en-us/dotnet/api/system.datetime.ticks?view=net-6.0#remarks).

Most of the time in MCU world, it doesn't make sense to keep track on time in such high resolution because 1) it's expensive (memory and update instructions) 2) there is no real need 3) it's hard to do anything useful with soooo much details, because the CPU can't handle it

Reply:
Thanks a lot for the details.
Another thing I love with .Net NF: the amount of things I learned in such a short time period !